src/webui/gitutils.jl

```diff
@@ -0,0 +1,159 @@
+# # Run some GitForge function, warning on error but still returning the value.
+macro gf(ex::Expr)
+    quote
+        let result = $(esc(ex))
+            GitForge.exception(result) === nothing ||
+                @warn "API request failed" exception=GitForge.exception(result)
+            GitForge.value(result)
```

Comment:
```suggestion
            return GitForge.value(result)
```

Reply:
I do not think we want this. 
```julia
function foo()
    x = @gf get_user(gh)
    return x.login
```
This code will return `x`, the last line will never be reached.

Reply:
ah, right, I forgot it was a macro.